css/index.css

```diff
@@ -112,10 +114,119 @@ p {
     justify-content: center;
 }
 
```

Comment:
One of the good things about CSS is for each box below you could get rid of 7 lines of code per box and let the parent .boxes or .box contain the styles except for the color, which you would want to override for each box.  See my comments in the index.html